eno-core/src/main/java/fr/insee/eno/core/processing/impl/DDIInsertControls.java

```diff
@@ -21,47 +21,47 @@ public void apply(EnoQuestionnaire enoQuestionnaire) { // Note: code is a bit cl
         assert enoQuestionnaire.getIndex() != null;
         //
         for (Sequence sequence : enoQuestionnaire.getSequences()) {
-            List<SequenceItem> sequenceItems = sequence.getSequenceItems();
+            List<ItemReference> sequenceItems = sequence.getSequenceItems();
             if (! sequenceItems.isEmpty()) {
                 int bound = sequenceItems.size();
```

Comment:
may be a little more semantic

Reply:
This one should also be refactored and tested